notebooks/agent_with_human_in_the_loop.ipynb

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
<p>I think we can use <code>pip install haystack-ai&gt;=2.13</code> here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>